modules/logging_configuration/main.tf

```diff
@@ -0,0 +1,37 @@
+resource "aws_verifiedaccess_instance_logging_configuration" "this" {
+
+  count = var.enable_logging ? 1 : 0
+
+  access_logs {
+    cloudwatch_logs {
+      enabled   = var.cloudwatch_logs.enable
+      log_group = aws_cloudwatch_log_group.cloudwatch_log_group[0].arn
```

Comment:
@franklinpashok should we give an option to use an existing log group. Similar to other two options.